src/Core/Repositories/TableStorage/EventRepository.cs

```diff
@@ -78,9 +80,9 @@ public async Task CreateAsync(IEvent e)
         await CreateEventAsync(entity);
     }
 
-    public async Task CreateManyAsync(IEnumerable<IEvent> e)
+    public async Task CreateManyAsync(IEnumerable<IEvent>? e)
     {
-        if (!e?.Any() ?? true)
+        if (e is null || !e.Any())
```

Comment:
The old pattern confuses null state analysis and makes it think that e is possibly null even though it shouldn't. This way of writing it helps it understand and in my opinion is more readable. We likely don't even want to allow null to be passed in but as this method is currently written it is meant to gracefully handle null so I kept it. When I go through the service layer I may revisit this to disallow a null collection.

Reply:
I have to agree that it's way more readable like this, the previous way made me pull out pen and paper to evaluate the order of operations lol